designs/README.md

```diff
@@ -1,25 +1,22 @@
-# Designs, Their Configuration, useful utilties:
-
-
-## Adding a design
-
+# The OpenLane Designs Folder
 To add a new design, the following command creates a configuration file for your design:
 
 ```bash
-./flow.tcl -design <design_name> -init_design_config
+# JSON Configuration File
+./flow.tcl -design <design_name> -init_design_config -add_to_designs
+
+# Tcl Configuration File
+./flow.tcl -design <design_name> -init_design_config -add_to_designs -config_file config.tcl
 ```
 
 This will create the following directory structure:
 
 ```bash
 designs/<design_name>
-├── config.tcl
+├── config.json (or config.tcl)
 ```
-In the configuration file, you should edit the required variables and the optional variables, if needed. Further information about the variables can be found [here][2]
-
-Also, the <design_name> could be  replaced by the <design_directory>, which will allow you to run any design on your machine.
-
-**Note: config.tcl is a global configuration for all PDKs. For more information about design configuration files please visit this [section](#configuration-files)**
+In the configuration file (`config.json`), you should edit the required variables and the optional variables, if needed. Further information about the variables can be found [here][2].
```

Comment:
Not due to this PR change but `[here][2]` is broken

Reply:
Fixed